aws/aws-how-to/kubernetes/deploy-ubuntu-cluster-with-eks-ami.rst

```diff
@@ -0,0 +1,65 @@
+Deploy an Ubuntu EKS cluster
+============================
+
+This guide shows how to deploy an Ubuntu EKS cluster using an EKS AMI.
+
+Prerequisites
+-------------
+
+You need:
+
+- ``eksctl`` (version 0.190 or newer): Check the instructions to `install eksctl`_
+- ``kubectl``: Check the instructions to `install kubectl`_
+
+
+Create the ``eksctl`` config file
+---------------------------------
+
+Create a ``config.yaml`` with the following content:
+
+..  code-block:: yaml
+
+    apiVersion: eksctl.io/v1alpha5
+    kind: ClusterConfig
+    metadata:
+        name: your-cluster
+        region: us-east-1
+    nodeGroups:
+    - name: ng-cluster
+      instanceType: m5.large
+      amiFamily: Ubuntu2204
+      ami: ami-ubuntu
+      overrideBootstrapCommand: |
+        #!/bin/bash
+        /etc/eks/bootstrap.sh <cluster-name>
```

Comment:
is <cluster-name> automatically substituted or does this need to duplicated?

Reply:
It's not automatically passed. This needs to be configured in the file to use custom AMI.

Reply:
for consistency, i would make sure this matches the cluster-name supplied earlier in the file.